Scripts/build_all.sh

```diff
@@ -49,34 +49,22 @@ main() {
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme OptimizelySDKUserProfileServiceTVOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelyiOSDemoApp -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKCoreiOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKCoreiOSTests -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKCoreTVOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKCoreTVOSTests -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKDatafileManageriOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKDatafileManageriOSTests -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKDatafileManagerTVOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKDatafileManagerTVOSTests -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKEventDispatcheriOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKEventDispatcheriOSTests -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKEventDispatcherTVOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKEventDispatcherTVOSTests -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKiOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKiOSTests -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKSharediOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKSharediOSTests -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKSharedTVOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKSharedTVOSTests -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKTVOS -configuration Release "${action}"
   xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKTVOSTests -configuration Release "${action}"
-  xcodebuild -workspace OptimizelySDK.xcworkspace -scheme Pods-OptimizelySDKUserProfileServiceiOS -configuration Release "${action}"
```

Comment:
why did we remove these build commands from the build-all script?

Reply:
"pod install" and "pod update" automatically calculate which Pods-Xxx are added to the workspace.  We've done work which affects that calculation, and with fewer autogenerated Pods-Xxx coming from "pod", our "build_all.sh" can be smaller.

Many of the "non tests" Xxx targets no longer depend on any pods and previously may have depended on FMDB and/or JSONModel . Those Xxx targets which no longer depend on pods have their Pods-Xxx go away since these CocoaPods automatically generated targets no longer exist.